src/containers/Page/components/Footer.tsx

```diff
@@ -6,187 +6,219 @@
-  margin: 0;
-  padding: 0;
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.xsmall};
-`;
-
-const StyledSafeLink = styled(SafeLink)`
-  color: white;
-  box-shadow: none;
-  text-decoration: underline;
-  text-underline-offset: 5px;
-  &:hover,
-  &:focus,
-  &:active {
-    text-decoration: none;
-  }
-`;
-
-const LinkListElement = styled.li`
-  display: flex;
-  align-items: center;
-  gap: ${spacing.xsmall};
-  padding: 0px;
-  svg {
-    width: 20px;
-    height: 20px;
-  }
-`;
+// TODO: This does not fully adhere to the design. The links switch positions at smaller breakpoints in the design, whereas they remain as-is here.
+// Let's keep things in their natural order for now.
+
+export const FooterBlock = styled("footer", {
+  base: {
+    position: "relative",
+    background: "primary",
+    color: "text.onAction",
+  },
+});
+
+export const StyledOneColumn = styled(OneColumn, {
+  base: {
+    position: "relative",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "space-between",
+    paddingBlock: "xxlarge",
+    tablet: {
+      paddingBlock: "4xlarge",
+    },
+    desktopDown: {
+      flexDirection: "column-reverse",
+      gap: "xxlarge",
+    },
+    gap: "4xlarge",
+    "& a:focus-visible": {
+      outlineColor: "surface.default",
+    },
+  },
+});
+
+const FooterTextWrapper = styled("div", {
+  base: {
+    tabletWide: {
+      alignSelf: "flex-end",
+    },
+    desktop: {
+      gridColumn: "span 2",
+    },
+  },
+});
+
+const StyledZendesk = styled(ZendeskButton, {
+  base: {
+    position: "absolute",
+    right: "xxlarge",
+    top: "-medium",
+    zIndex: "docked",
+  },
+});
+
+const StyledLinkBlock = styled("div", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "3xsmall",
+  },
+});
+
+const LinkList = styled("ul", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "3xsmall",
+  },
+});
+
+const StyledSafeLink = styled(SafeLink, {
+  base: {
+    display: "inline",
+    color: "text.onAction",
+    textStyle: "body.link",
+    textDecoration: "underline",
+    "& svg": {
+      marginInlineStart: "xsmall",
+    },
+    _hover: {
+      textDecoration: "none",
+    },
+    _focusVisible: {
+      textDecoration: "none",
+    },
+  },
+});
 
 interface FooterLinkBlockProps {
-  links: { to: string; text: string; external?: boolean; icon?: ReactNode; asAnchor?: boolean }[];
+  links: { to: string; text: string }[];
   label: string;
-  className?: string;
 }
 
-const FooterLinkBlock = ({ links, label, className }: FooterLinkBlockProps) => {
+const FooterLinkBlock = ({ links, label }: FooterLinkBlockProps) => {
   const id = useId();
   return (
-    <StyledLinkBlock className={className}>
-      <Heading id={id} element="span" headingStyle="list-title" margin="none">
-        {label}
+    <StyledLinkBlock>
+      <Heading id={id} asChild consumeCss textStyle="label.large" fontWeight="bold">
+        <span>{label}</span>
```

Comment:
Hvorfor span? (gjelder de andre headingene og)

Reply:
Vi fikk tyn en gang i tiden for å ha headings i footer. Følger bare slik det var før.

Reply:
Hmm, husker du om det var noen spesiell grunn til det? Jeg stemmer hvertfall for å bruke riktige elementer i ny footer 🤔

Reply:
Tror det var en i-greie for de. Får Google litt!